inginious/frontend/task_dispensers/util.py

```diff
@@ -212,6 +212,17 @@ def __init__(self, structure):
                 else:
                     raise InvalidTocException( ("The weight value must be a numeric >= 0 for the task: " + str(taskid)) )
 
+        self._store_submission = {}
```

Comment:
I guess this piece of code was taken from the edit task page code, but I don't see it removed from anywhere  in the code in this PR.